spec/requests/api/v0/user_tattoos/delete_user_tattoos_request_spec.rb

```diff
@@ -0,0 +1,49 @@
+require "rails_helper"
+
+RSpec.describe "endpoint delete /api/v0/user_tattoos" do
+  describe "As a User" do
+
+    before do
+      @user = create(:user)
+      @id = @user.id
+
+      @artist = create(:artist)
+
+      @tattoo = Tattoo.create!({artist_id: @artist.id, price: 250, time_estimate: 90, image_url: "image/path"})
+
+      @params = {
+        user_id: @id,
+        tattoo_id: @tattoo.id
+      }
+
+      @headers = {
+        "CONTENT_TYPE": "application/json",
+        "ACCEPT": "application/json"
+      }
+
+      @user_tattoo = UserTattoo.create!(@params)
+    end
+
+    it "deletes a user tattoo relation via HTTP request" do
+        delete "/api/v0/user_tattoos", params: JSON.generate(user_tattoo: @params), headers: @headers
+
+        expect(response).to be_successful
+        expect(response.status).to eq(204)
+    end
+
+    describe "Sad Path" do 
+      it "errors if association does not exist" do 
+        delete "/api/v0/user_tattoos", params: JSON.generate(user_tattoo: @params), headers: @headers
+
+        delete "/api/v0/user_tattoos", params: JSON.generate(user_tattoo: @params), headers: @headers
+        expect(response).not_to be_successful
+        expect(response.status).to eq(404)
+
+        error_response = JSON.parse(response.body, symbolize_names: true)
+
+        expect(error_response).to have_key(:message)
+        expect(error_response[:message]).to eq("Association between Tattoo and User does not exist")
```

Comment:
Looks great. We might be able to move some of this error handling into out ErrorMessage PORO and serializer, but I think that is a good refactor spot for us later so we can keep the ball rolling on getting endpoints prepared for the FE team. I'll add a note for us to refer back to this spot later when we get to a refactor point.